.github/workflows/build-images-base-v1.17.yaml

```diff
@@ -2,14 +2,6 @@ name: Base Image Release Build
 
 # Any change in triggers needs to be reflected in the concurrency group.
 on:
-  pull_request_target:
```

Comment:
Should we also drop the `pull_request_target` condition check from the `build-and-push` job?